### ISSUE TITLE
[Issue]: NVENC not working after 10.7.7 -> 10.8.0 upgrade

### ISSUE THREAD

**Issue by peperunas** [jellyfin/jellyfin#7944]
### Please describe your bug

FFmpeg fails to encode a media file with hardware acceleration on.

Jellyfin on NixOS build, `master` rev `b1957596ff1c7aa8c55c4512b7ad1c9672502e8e`.

`jellyfin-ffmpeg`: `5.0.1`
`jellyfin`: `1.8.0`
`jellyfin-web`: `1.8.0`

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: NixOS
- Virtualization: None
- Clients: None
- Browser: Firefox
- FFmpeg Version:

$ /nix/store/wadg8z2n62wim3m72ffmh1xfgbyjwzpb-jellyfin-ffmpeg-5.0.1-5/bin/ffmpeg   
ffmpeg version 5.0.1 Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 11.3.0 (GCC)
  configuration: --disable-static --prefix=/nix/store/wadg8z2n62wim3m72ffmh1xfgbyjwzpb-jellyfin-ffmpeg-5.0.1-5 --target_os=linux --arch=x86_64 --enable-gpl --enable-version3 --disable-nonfree --enable-shared --enable-pic --disable-small --enable-runtime-cpudetect --disable-lto --enable-gray --enable-swscale-alpha --enable-hardcoded-tables --enable-safe-bitstream-reader --enable-pthreads --disable-w32threads --disable-os2threads --enable-network --enable-pixelutils --enable-ffmpeg --enable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avfilter --enable-avformat --enable-avutil --enable-postproc --enable-swresample --enable-swscale --enable-doc --disable-htmlpages --enable-manpages --disable-podpages --disable-txtpages --enable-bzlib --enable-libcelt --enable-libdav1d --disable-libfdk-aac --disable-libflite --enable-fontconfig --enable-libfreetype --enable-frei0r --enable-libfribidi --enable-libgme --enable-gnutls --enable-libgsm --enable-ladspa --enable-libmp3lame --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libdc1394 --enable-libdrm --enable-iconv --enable-libjack --enable-libmfx --enable-libmodplug --enable-libmysofa --enable-libopus --enable-librsvg --enable-libsrt --enable-libssh --enable-libtheora --enable-libv4l2 --enable-vaapi --enable-vdpau --enable-libvorbis --enable-libvmaf --enable-libvpx --enable-libwebp --enable-xlib --enable-libxcb --enable-libxcb-shm --enable-libxcb-xfixes --enable-libxcb-shape --enable-libxml2 --enable-lzma --enable-nvenc --enable-openal --enable-libopencore-amrnb --disable-opengl --enable-libopenh264 --enable-libopenjpeg --disable-openssl --enable-libpulse --disable-librav1e --enable-libsvtav1 --disable-librtmp --enable-sdl2 --enable-libsoxr --enable-libspeex --enable-libvidstab --enable-libvo-amrwbenc --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzmq --enable-libzimg --enable-zlib --enable-vulkan --enable-libglslang --enable-libsmbclient --disable-debug --enable-optimizations --disable-extra-warnings --disable-stripping
  libavutil      57. 17.100 / 57. 17.100
  libavcodec     59. 18.100 / 59. 18.100
  libavformat    59. 16.100 / 59. 16.100
  libavdevice    59.  4.100 / 59.  4.100
  libavfilter     8. 24.100 /  8. 24.100
  libswscale      6.  4.100 /  6.  4.100
  libswresample   4.  3.100 /  4.  3.100
  libpostproc    56.  3.100 / 56.  3.100
Hyper fast Audio and Video encoder
```
- Playback Method: Transcoding
- Hardware Acceleration: NVENC
- Plugins:
- Reverse Proxy: nginx
- Base URL:
- Networking:
- Storage:
```


### Jellyfin logs

_No response_

### FFmpeg logs

```shell
/videos/6948e3d0-04e7-ae69-757b-5ba0e06b967d/hls1/main/0.ts

{"Protocol":0,"Id":"6948e3d004e7ae69757b5ba0e06b967d","Path":"/media/TVShows/x/Season 4/x - y - Winner Bluray-1080p Proper.mkv","EncoderPath":null,"EncoderProtocol":null,"Type":0,"Container":"mkv,webm","Size":4693431808,"Name":"x - y - Winner Bluray-1080p Proper","IsRemote":false,"ETag":"ac8f51ea11b55fe3392168a681bf785e","RunTimeTicks":36046077952,"ReadAtNativeFramerate":false,"IgnoreDts":false,"IgnoreIndex":false,"GenPtsInput":false,"SupportsTranscoding":true,"SupportsDirectStream":true,"SupportsDirectPlay":true,"IsInfiniteStream":false,"RequiresOpening":false,"OpenToken":null,"RequiresClosing":false,"LiveStreamId":null,"BufferMs":null,"RequiresLooping":false,"SupportsProbing":true,"VideoType":0,"IsoType":null,"Video3DFormat":null,"MediaStreams":[{"Codec":"subrip","CodecTag":null,"Language":"per","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/1000","CodecTimeBase":null,"Title":null,"VideoRange":null,"LocalizedUndefined":"Undefined","LocalizedDefault":"Default","LocalizedForced":"Forced","LocalizedExternal":"External","DisplayTitle":"Per - SUBRIP - External","NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":null,"BitRate":null,"BitDepth":null,"RefFrames":null,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":false,"IsForced":false,"Height":null,"Width":null,"AverageFrameRate":null,"RealFrameRate":null,"Profile":null,"Type":2,"AspectRatio":null,"Index":0,"Score":null,"IsExternal":true,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":true,"SupportsExternalStream":true,"Path":"/media/TVShows/x/Season 4/x - y - Winner Bluray-1080p Proper.fa.srt","PixelFormat":null,"Level":0,"IsAnamorphic":null},{"Codec":"subrip","CodecTag":null,"Language":"eng","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/1000","CodecTimeBase":null,"Title":null,"VideoRange":null,"LocalizedUndefined":"Undefined","LocalizedDefault":"Default","LocalizedForced":"Forced","LocalizedExternal":"External","DisplayTitle":"English - SUBRIP - External","NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":null,"BitRate":null,"BitDepth":null,"RefFrames":null,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":false,"IsForced":false,"Height":null,"Width":null,"AverageFrameRate":null,"RealFrameRate":null,"Profile":null,"Type":2,"AspectRatio":null,"Index":1,"Score":null,"IsExternal":true,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":true,"SupportsExternalStream":true,"Path":"/media/TVShows/x/Season 4/x - y - Winner Bluray-1080p Proper.en.srt","PixelFormat":null,"Level":0,"IsAnamorphic":null},{"Codec":"subrip","CodecTag":null,"Language":"ita","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/1000","CodecTimeBase":null,"Title":null,"VideoRange":null,"LocalizedUndefined":"Undefined","LocalizedDefault":"Default","LocalizedForced":"Forced","LocalizedExternal":"External","DisplayTitle":"Italian - SUBRIP - External","NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":null,"BitRate":null,"BitDepth":null,"RefFrames":null,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":false,"IsForced":false,"Height":null,"Width":null,"AverageFrameRate":null,"RealFrameRate":null,"Profile":null,"Type":2,"AspectRatio":null,"Index":2,"Score":null,"IsExternal":true,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":true,"SupportsExternalStream":true,"Path":"/media/TVShows/x/Season 4/x - y - Winner Bluray-1080p Proper.it.srt","PixelFormat":null,"Level":0,"IsAnamorphic":null},{"Codec":"h264","CodecTag":null,"Language":"eng","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/1000","CodecTimeBase":null,"Title":null,"VideoRange":"SDR","LocalizedUndefined":null,"LocalizedDefault":null,"LocalizedForced":null,"LocalizedExternal":null,"DisplayTitle":"1080p H264 SDR","NalLengthSize":"0","IsInterlaced":false,"IsAVC":false,"ChannelLayout":null,"BitRate":10416515,"BitDepth":8,"RefFrames":1,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":true,"IsForced":false,"Height":1080,"Width":1920,"AverageFrameRate":23.976025,"RealFrameRate":23.976025,"Profile":"High","Type":1,"AspectRatio":"16:9","Index":3,"Score":null,"IsExternal":false,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Path":null,"PixelFormat":"yuv420p","Level":41,"IsAnamorphic":null},{"Codec":"dts","CodecTag":null,"Language":"eng","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/1000","CodecTimeBase":null,"Title":null,"VideoRange":null,"LocalizedUndefined":null,"LocalizedDefault":null,"LocalizedForced":null,"LocalizedExternal":null,"DisplayTitle":"English - DTS - 5.1 - Default","NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":"5.1","BitRate":1536000,"BitDepth":null,"RefFrames":null,"PacketLength":null,"Channels":6,"SampleRate":48000,"IsDefault":true,"IsForced":false,"Height":null,"Width":null,"AverageFrameRate":null,"RealFrameRate":null,"Profile":"DTS","Type":0,"AspectRatio":null,"Index":4,"Score":null,"IsExternal":false,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Path":null,"PixelFormat":null,"Level":0,"IsAnamorphic":null},{"Codec":"subrip","CodecTag":null,"Language":"eng","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/1000","CodecTimeBase":null,"Title":"Forced","VideoRange":null,"LocalizedUndefined":"Undefined","LocalizedDefault":"Default","LocalizedForced":"Forced","LocalizedExternal":"External","DisplayTitle":"Forced - English - Default - SUBRIP","NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":null,"BitRate":null,"BitDepth":null,"RefFrames":null,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":true,"IsForced":true,"Height":null,"Width":null,"AverageFrameRate":null,"RealFrameRate":null,"Profile":null,"Type":2,"AspectRatio":null,"Index":5,"Score":null,"IsExternal":false,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":true,"SupportsExternalStream":true,"Path":null,"PixelFormat":null,"Level":0,"IsAnamorphic":null}],"MediaAttachments":[],"Formats":[],"Bitrate":11952515,"Timestamp":null,"RequiredHttpHeaders":{},"TranscodingUrl":null,"TranscodingSubProtocol":null,"TranscodingContainer":null,"AnalyzeDurationMs":null,"DefaultAudioStreamIndex":null,"DefaultSubtitleStreamIndex":null}

/nix/store/wadg8z2n62wim3m72ffmh1xfgbyjwzpb-jellyfin-ffmpeg-5.0.1-5/bin/ffmpeg -analyzeduration 200M  -i file:"/media/TVShows/x/Season 4/x - y - Winner Bluray-1080p Proper.mkv" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:0 -map 0:1 -map -0:s -codec:v:0 h264_nvenc -preset p4 -b:v 8393430 -maxrate 8393430 -bufsize 16786860 -profile:v:0 high -g:v:0 72 -keyint_min:v:0 72 -vf "setparams=color_primaries=bt709:color_trc=bt709:colorspace=bt709,scale=trunc(min(max(iw\,ih*a)\,1920)/2)*2:trunc(ow/a/2)*2,format=yuv420p" -codec:a:0 aac -ac 2 -ab 384000 -af "volume=2" -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 0 -hls_segment_filename "/jellyfin/transcodes/ebbb600e563350f80367ba87ce1de647%d.ts" -hls_playlist_type vod -hls_list_size 0 -y "/jellyfin/transcodes/ebbb600e563350f80367ba87ce1de647.m3u8"


ffmpeg version 5.0.1 Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 11.3.0 (GCC)
  configuration: --disable-static --prefix=/nix/store/wadg8z2n62wim3m72ffmh1xfgbyjwzpb-jellyfin-ffmpeg-5.0.1-5 --target_os=linux --arch=x86_64 --enable-gpl --enable-version3 --disable-nonfree --enable-shared --enable-pic --disable-small --enable-runtime-cpudetect --disable-lto --enable-gray --enable-swscale-alpha --enable-hardcoded-tables --enable-safe-bitstream-reader --enable-pthreads --disable-w32threads --disable-os2threads --enable-network --enable-pixelutils --enable-ffmpeg --enable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avfilter --enable-avformat --enable-avutil --enable-postproc --enable-swresample --enable-swscale --enable-doc --disable-htmlpages --enable-manpages --disable-podpages --disable-txtpages --enable-bzlib --enable-libcelt --enable-libdav1d --disable-libfdk-aac --disable-libflite --enable-fontconfig --enable-libfreetype --enable-frei0r --enable-libfribidi --enable-libgme --enable-gnutls --enable-libgsm --enable-ladspa --enable-libmp3lame --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libdc1394 --enable-libdrm --enable-iconv --enable-libjack --enable-libmfx --enable-libmodplug --enable-libmysofa --enable-libopus --enable-librsvg --enable-libsrt --enable-libssh --enable-libtheora --enable-libv4l2 --enable-vaapi --enable-vdpau --enable-libvorbis --enable-libvmaf --enable-libvpx --enable-libwebp --enable-xlib --enable-libxcb --enable-libxcb-shm --enable-libxcb-xfixes --enable-libxcb-shape --enable-libxml2 --enable-lzma --enable-nvenc --enable-openal --enable-libopencore-amrnb --disable-opengl --enable-libopenh264 --enable-libopenjpeg --disable-openssl --enable-libpulse --disable-librav1e --enable-libsvtav1 --disable-librtmp --enable-sdl2 --enable-libsoxr --enable-libspeex --enable-libvidstab --enable-libvo-amrwbenc --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzmq --enable-libzimg --enable-zlib --enable-vulkan --enable-libglslang --enable-libsmbclient --disable-debug --enable-optimizations --disable-extra-warnings --disable-stripping
  libavutil      57. 17.100 / 57. 17.100
  libavcodec     59. 18.100 / 59. 18.100
  libavformat    59. 16.100 / 59. 16.100
  libavdevice    59.  4.100 / 59.  4.100
  libavfilter     8. 24.100 /  8. 24.100
  libswscale      6.  4.100 /  6.  4.100
  libswresample   4.  3.100 /  4.  3.100
  libpostproc    56.  3.100 / 56.  3.100
Input #0, matroska,webm, from 'file:/media/TVShows/x/Season 4/x - y - Winner Bluray-1080p Proper.mkv':
  Metadata:
    encoder         : libebml v1.3.1 + libmatroska v1.4.2
    creation_time   : 2019-04-29T00:41:27.000000Z
  Duration: 01:00:04.61, start: 0.000000, bitrate: 10416 kb/s
  Chapters:
    Chapter #0:0: start 0.000000, end 455.830000
      Metadata:
        title           : 00:00:00.000
    Chapter #0:1: start 455.830000, end 1129.920000
      Metadata:
        title           : 00:07:35.830
    Chapter #0:2: start 1129.920000, end 1575.073000
      Metadata:
        title           : 00:18:49.920
    Chapter #0:3: start 1575.073000, end 2304.010000
      Metadata:
        title           : 00:26:15.073
    Chapter #0:4: start 2304.010000, end 2971.969000
      Metadata:
        title           : 00:38:24.010
    Chapter #0:5: start 2971.969000, end 3559.890000
      Metadata:
        title           : 00:49:31.969
    Chapter #0:6: start 3559.890000, end 3604.608000
      Metadata:
        title           : 00:59:19.890
  Stream #0:0(eng): Video: h264 (High), yuv420p(progressive), 1920x1080, SAR 1:1 DAR 16:9, 23.98 fps, 23.98 tbr, 1k tbn (default)
    Metadata:
      BPS             : 8905106
      BPS-eng         : 8905106
      DURATION        : 01:00:04.601000000
      DURATION-eng    : 01:00:04.601000000
      NUMBER_OF_FRAMES: 86424
      NUMBER_OF_FRAMES-eng: 86424
      NUMBER_OF_BYTES : 4012419681
      NUMBER_OF_BYTES-eng: 4012419681
      _STATISTICS_WRITING_APP: mkvmerge v8.3.0 ('Over the Horizon') 64bit
      _STATISTICS_WRITING_APP-eng: mkvmerge v8.3.0 ('Over the Horizon') 64bit
      _STATISTICS_WRITING_DATE_UTC: 2019-04-29 00:41:27
      _STATISTICS_WRITING_DATE_UTC-eng: 2019-04-29 00:41:27
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
  Stream #0:1(eng): Audio: dts (DTS), 48000 Hz, 5.1(side), fltp, 1536 kb/s (default)
    Metadata:
      BPS             : 1509000
      BPS-eng         : 1509000
      DURATION        : 01:00:04.608000000
      DURATION-eng    : 01:00:04.608000000
      NUMBER_OF_FRAMES: 337932
      NUMBER_OF_FRAMES-eng: 337932
      NUMBER_OF_BYTES : 679919184
      NUMBER_OF_BYTES-eng: 679919184
      _STATISTICS_WRITING_APP: mkvmerge v8.3.0 ('Over the Horizon') 64bit
      _STATISTICS_WRITING_APP-eng: mkvmerge v8.3.0 ('Over the Horizon') 64bit
      _STATISTICS_WRITING_DATE_UTC: 2019-04-29 00:41:27
      _STATISTICS_WRITING_DATE_UTC-eng: 2019-04-29 00:41:27
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
  Stream #0:2(eng): Subtitle: subrip (default) (forced)
    Metadata:
      title           : Forced
      BPS             : 7
      BPS-eng         : 7
      DURATION        : 00:10:16.574000000
      DURATION-eng    : 00:10:16.574000000
      NUMBER_OF_FRAMES: 21
      NUMBER_OF_FRAMES-eng: 21
      NUMBER_OF_BYTES : 606
      NUMBER_OF_BYTES-eng: 606
      _STATISTICS_WRITING_APP: mkvmerge v8.3.0 ('Over the Horizon') 64bit
      _STATISTICS_WRITING_APP-eng: mkvmerge v8.3.0 ('Over the Horizon') 64bit
      _STATISTICS_WRITING_DATE_UTC: 2019-04-29 00:41:27
      _STATISTICS_WRITING_DATE_UTC-eng: 2019-04-29 00:41:27
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> h264 (h264_nvenc))
  Stream #0:1 -> #0:1 (dts (dca) -> aac (native))
Press [q] to stop, [?] for help
[h264_nvenc @ 0x91ff40] [Eval @ 0x7fffaa324b70] Undefined constant or missing '(' in 'p4'
[h264_nvenc @ 0x91ff40] Unable to parse option value "p4"
[h264_nvenc @ 0x91ff40] Error setting option preset to value p4.
Error initializing output stream 0:0 -- Error while opening encoder for output stream #0:0 - maybe incorrect parameters such as bit_rate, rate, width or height
[aac @ 0xa7e9c0] Qavg: 55094.492
[aac @ 0xa7e9c0] 2 frames left in the queue on closing
Conversion failed!
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nyanmisaka** [comment]
The ffnvcodec header of NixOS's custom ffmpeg is outdated.
https://github.com/FFmpeg/FFmpeg/blob/a10f1aec1fe59ff3aee3fb93be44142ba33a5c1d/libavcodec/nvenc.h#L65-L67

Also ffmpeg needs to be built with following options for Nvidia to keep in line with our jellyfin-ffmpeg5:
https://github.com/jellyfin/jellyfin-ffmpeg/blob/jellyfin/debian/rules
```
	--disable-ptx-compression \
	--enable-ffnvcodec \
	--enable-cuda \
	--enable-cuda-llvm \
	--enable-cuvid \
	--enable-nvdec \
	--enable-nvenc \
```

**Comment by justinas** [comment]
> The [ffnvcodec](https://github.com/FFmpeg/nv-codec-headers)header of NixOS's custom ffmpeg is outdated.

There are versions 10 and 11 in Nixpkgs (if I'm looking at the right thing), but the default is 9. https://search.nixos.org/packages?channel=22.05&from=0&size=50&sort=relevance&type=packages&query=nvenc

Should we override `jellyfin-ffmpeg` to use version 10?

I'll look into adding the other flags in the coming days.

**Comment by nyanmisaka** [comment]
> > The [ffnvcodec](https://github.com/FFmpeg/nv-codec-headers) header of NixOS's custom ffmpeg is outdated.
> 
> There are versions 10 and 11 in Nixpkgs (if I'm looking at the right thing), but the default is 9. https://search.nixos.org/packages?channel=22.05&from=0&size=50&sort=relevance&type=packages&query=nvenc
> 
> Should we override `jellyfin-ffmpeg` to use version 10?
> 
> I'll look into adding the other flags in the coming days.

We are using 11 in jellyfin-ffmpeg5, and the required minimum driver version is 470.

Also please add `--disable-ptx-compression` to build configs, which is required by our cuda tonemap filter.

**Comment by justinas** [comment]
I have created 3 PRs to Nixpkgs:

1. https://github.com/NixOS/nixpkgs/pull/178276 - This is the simplest, least contentious one and should fix the root cause of failing NVENC transcoding.
2. https://github.com/NixOS/nixpkgs/pull/178283 - HW decoding support
3. https://github.com/NixOS/nixpkgs/pull/178278 - CUDA tonemap support

Thanks for your guidance. I understand that it sucks getting issue reports from people with "esoteric" setups :sweat_smile:

**Comment by nyanmisaka** [comment]
> I have created 3 PRs to Nixpkgs:
> 
> 1. [jellyfin-ffmpeg: bump nv-codec-headers to v11 NixOS/nixpkgs#178276](https://github.com/NixOS/nixpkgs/pull/178276) - This is the simplest, least contentious one and should fix the root cause of failing NVENC transcoding.
> 2. [ffmpeg-full: add NVDEC support NixOS/nixpkgs#178283](https://github.com/NixOS/nixpkgs/pull/178283) - HW decoding support
> 3. [ffmpeg-full: add cuda features, jellyfin-ffmpeg: fix CUDA tonemap NixOS/nixpkgs#178278](https://github.com/NixOS/nixpkgs/pull/178278) - CUDA tonemap support
> 
> Thanks for your guidance. I understand that it sucks getting issue reports from people with "esoteric" setups 😅

Thanks! Any chance you can also add `--enable-opencl`, which is required for Intel and AMD to do HW tonemap.

**Comment by justinas** [comment]
> Thanks! Any chance you can also add `--enable-opencl`, which is required for Intel and AMD to do HW tonemap.

Already open https://github.com/NixOS/nixpkgs/pull/177438
